src/main/java/com/ace/alfox/game/models/Player.java

```diff
@@ -14,6 +16,10 @@
   public Vector2 location = new Vector2(0, 0);
   public PlayerState playerState = PlayerState.FREE;
   public Long activeBattleId = null;
+  @JsonIgnore
+  public byte[] password = new byte[0]; // Neither of these should be in player, they are user attributes, but no user class exists as of time of addition
```

Comment:
Will this get serialized and sent with the request now? Or does that not happen because of @JsonIgnore?

Reply:
Does not get serialized because of the JsonIgnore.